website/content/specification/syntax/constraints.md

```diff
@@ -14,27 +14,26 @@ TODO: P3: Address issue https://github.com/usnistgov/metaschema/issues/325
 
 ## Enumerated values
 
-Additionally, flags may be constrained to a set of known values listed in advance.
+One kind of Metaschema constraint is restricting field, or flag value(s) with `allowed-values` enumerations. Metaschema processors MUST process `allowed-values` enumerations.
```

Comment:
We need to discuss the targeting of constraints using `@target`, which can be done generically for all constraints. Some types of constraints have an implicit '.' context target, such as those directly on a flag, since a flag has no children.